<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Cyphics</author>
 <class>UniverseIdleMainWindow</class>
 <widget class="QMainWindow" name="UniverseIdleMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>917</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Universe Idle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="mainGroup">
      <property name="title">
       <string>main group</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>1051</width>
         <height>721</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="displayLayout">
        <item>
         <widget class="QGroupBox" name="displayGroup">
          <property name="title">
           <string>display Group</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>29</y>
             <width>591</width>
             <height>681</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="infoLayout">
            <item>
             <widget class="QGroupBox" name="infoGroup">
              <property name="title">
               <string>infoBox</string>
              </property>
              <widget class="QWidget" name="formLayoutWidget">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>240</y>
                 <width>441</width>
                 <height>81</height>
                </rect>
               </property>
               <layout class="QFormLayout" name="physics_layout">
                <item row="0" column="0">
                 <widget class="QLabel" name="speed_label">
                  <property name="text">
                   <string>Speed :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="speed_value_label">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="distance_label">
                  <property name="text">
                   <string>Distance traveled :</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="distance_value_label">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="acceleration_label">
                  <property name="text">
                   <string>Acceleration : </string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="time_label">
                  <property name="text">
                   <string>Time until destination :</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="time_value_label">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="acceleration_value_label">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="formLayoutWidget">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>340</y>
                 <width>291</width>
                 <height>31</height>
                </rect>
               </property>
               <layout class="QFormLayout" name="resources_layout">
                <item row="0" column="0">
                 <widget class="QLabel" name="kinetic_energy_label">
                  <property name="text">
                   <string>Cinetic energy :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="kinetic_energy_value_label">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QTextBrowser" name="textBrowser">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>390</y>
                 <width>531</width>
                 <height>271</height>
                </rect>
               </property>
              </widget>
              <widget class="QWidget" name="horizontalLayoutWidget">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>40</y>
                 <width>531</width>
                 <height>31</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="speed_label_2">
                  <property name="text">
                   <string>Speed :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="speed_value_label_2">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="speed_combo_box"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="horizontalLayoutWidget_2">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>80</y>
                 <width>531</width>
                 <height>31</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="speed_label_3">
                  <property name="text">
                   <string>Time until destination:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="speed_value_label_3">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_2"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_6">
           <property name="geometry">
            <rect>
             <x>610</x>
             <y>30</y>
             <width>431</width>
             <height>191</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="increm_upgrades_layout">
            <item>
             <widget class="QGroupBox" name="IncremUpBox">
              <property name="title">
               <string>Incremental</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout"/>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_7">
           <property name="geometry">
            <rect>
             <x>610</x>
             <y>550</y>
             <width>431</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="science_layout">
            <item>
             <widget class="QGroupBox" name="science_box">
              <property name="title">
               <string>Science</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3"/>
             </widget>
            </item>
            <item>
             <spacer name="science_spacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_8">
           <property name="geometry">
            <rect>
             <x>610</x>
             <y>390</y>
             <width>431</width>
             <height>151</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="structures_layout">
            <item>
             <widget class="QGroupBox" name="structures_box">
              <property name="title">
               <string>Structures</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4"/>
             </widget>
            </item>
            <item>
             <spacer name="structures_spacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_9">
           <property name="geometry">
            <rect>
             <x>610</x>
             <y>230</y>
             <width>431</width>
             <height>151</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="upgrades_layout">
            <item>
             <widget class="QGroupBox" name="upgrades_box">
              <property name="title">
               <string>Upgrades</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5"/>
             </widget>
            </item>
            <item>
             <spacer name="upgrades_spacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>760</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="quitLayout"/>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>760</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="click_button_layout">
        <item>
         <widget class="QPushButton" name="click_button">
          <property name="text">
           <string>Click!</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
